packages/shared/web/src/header/launch-app-button/launch-app-button.tsx

```diff
@@ -1,17 +1,25 @@
 import { Button } from '@mui/material';
 import { useNotionalTheme } from '@notional-finance/styles';
 import { Network, THEME_VARIANTS } from '@notional-finance/util';
-import { Link } from 'react-router-dom';
+import { Link, useParams } from 'react-router-dom';
 import { FormattedMessage } from 'react-intl';
 import { LaunchIcon } from '@notional-finance/icons';
+import { useConnectWallet } from '@web3-onboard/react';
 
 export function LaunchAppButton() {
   const theme = useNotionalTheme(THEME_VARIANTS.LIGHT);
+  const params = useParams<any>();
+  const [{ wallet }] = useConnectWallet();
```

Comment:
Same here, we should wrap this into another hook that gets the first connected wallet.